### PR TITLE
BaSP-MR-154: Firebase Setup

### PR DESCRIPTION
**Pull request by NicolasKler**
I installed Firebase admin and rearrange the folders.

### CODE REVIEW

File: src/helper/firebase/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import * as admin from 'firebase-admin';
+
+require('dotenv').config();
+
+const firebaseApp = admin.initializeApp({
+  credential: admin.credential.cert({
+    project_id: process.env.FIREBASE_PROJECT_ID,
+    private_key_id: process.env.FIREBASE_PRIVATE_KEY,
+    private_key: process.env.FIREBASE_PRIVATE_KEY,
```
Comment (radiumlearning) [inline src/helper/firebase/index.js:9]:
```suggestion
    privateKey: process.env.FIREBASE_PRIVATE_KEY.replace(/\\n/g, '\n')
```

**Review by radiumlearning — CHANGES_REQUESTED** [review]

File: src/helper/firebase/index.js (hunk 1 above)
Comment (NicolasKler) [reply src/helper/firebase/index.js]: Done

File: src/helper/firebase/index.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import * as admin from 'firebase-admin';
+
+require('dotenv').config();
+
+const firebaseApp = admin.initializeApp({
+  credential: admin.credential.cert({
+    project_id: process.env.FIREBASE_PROJECT_ID,
+    private_key_id: process.env.FIREBASE_PRIVATE_KEY,
+    privateKey: process.env.PRIVATE_KEY_FIREBASE.replace(/\\n/g, '\n'),
```
Comment (julianv97) [inline src/helper/firebase/index.js:9]:
```suggestion
    private_key: process.env.FIREBASE_PRIVATE_KEY.replace(/\\n/g, '\n'),
```

File: src/helper/firebase/index.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import * as admin from 'firebase-admin';
+
+require('dotenv').config();
+
+const firebaseApp = admin.initializeApp({
+  credential: admin.credential.cert({
+    project_id: process.env.FIREBASE_PROJECT_ID,
+    private_key_id: process.env.FIREBASE_PRIVATE_KEY,
```
Comment (julianv97) [inline src/helper/firebase/index.js:8]:
```suggestion
    private_key_id: process.env.FIREBASE_PRIVATE_KEY_ID,
```

**Review by julianv97 — CHANGES_REQUESTED** [review]

**Review by radiumlearning — APPROVED** [review]

File: src/helper/firebase/index.js (hunk 3 above)
Comment (NicolasKler) [reply src/helper/firebase/index.js]: done

**Review by julianv97 — APPROVED** [review]

**Review by octavi01 — APPROVED** [review]

**Pull request merged**